src/GameShelf.js

```diff
@@ -39,7 +40,8 @@ export default class GameShelf extends Component {
       titleFilter: this.props.titleFilter,
       genres: this.props.genresToFilter,
       platforms: this.props.platformsToFilter,
-      isLucky: this.props.isLucky
+      isLucky: this.props.isLucky,
+      isMulti: this.props.multiplayerToggle
```

Comment:
This life cycle method is amazing and fundamental to making our app work, nice job updating it.